### ISSUE TITLE
Telegram is automatically started when I use a share button in antother app

### ISSUE THREAD

**Issue by mahdi1234** [Telegram-FOSS-Team/Telegram-FOSS#374]
Telegram is automatically started when I use a share button in another app even if I don't choose to share with Telegram app.

I can see that telegram is started because there is the notification icon that appears.

I don't know if there is a good reason that the app starts, but it stays opened after.

How to avoid that ?

There was case #269 closed as dupe of #270 but even after fixing #270 this is still problem. I find this behavior quite annoying as I don't want Telegram to start unless shared content with it.

**Comment by mahdi1234** [comment]
Disabling "Background connection" helps not to start-up Telegram when "Share via" appears, can you please explain a little this behavior why it actually does something before app to share with is actually chosen?

**Comment by danialbehzadi** [comment]
Did you try turning off the instant share?

**Comment by mahdi1234** [comment]
Yes I tried, but "Direct share" had no effect and still started the app.
